pkg/packagekit/updates.jsx

```diff
@@ -135,6 +135,8 @@ function deduplicate(list) {
 // Insert comma strings in between elements of the list. Unlike list.join(",")
 // this does not stringify the elements, which we need to keep as JSX objects.
 function insertCommas(list) {
+    if (list.length == 0)
+        return list;
     return list.reduce((prev, cur) => [prev, ", ", cur])
```

Comment:
This fix works, but note that there is another weird edge case in `Array.reduce` when not passing the `initialValue` parameter: for one-element lists, it doesn't call the callback and returns the element, not a list with that one element.  MDN documentation thus suggests always passing an `initalValue`.

Reply:
Good point. It doesn't matter here as JSX treats a single item and an one-element list the same, but for the sake of clarity I force-pushed a fix. There is no `initialValue` that actually works here, but just checking for `length <= 1` does the trick.

Reply:
Thanks!